lib/wibox/container/only_on_screen.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------
+--
```

Comment:
Yup. I think I copied this from the `mirror` container, but that one does have some example code (which is not really possible here).